tests/index.html

```diff
@@ -40,6 +40,15 @@
       (function() {
         window.EmberENV = window.EmberENV || {};
 
+        // Determine edition, and apply correct EmberENV settings
+        var edition = QUnit.urlParams.edition || 'octane';
+
+        if (edition === 'octane') {
+          EmberENV._TEMPLATE_ONLY_GLIMMER_COMPONENTS = true;
+          //EmberENV._APPLICATION_TEMPLATE_WRAPPER = false;
```

Comment:
This is commented out because it causes a test failure in this backtracking rerender test:

https://github.com/emberjs/ember.js/blob/cfb74eb8b83b70773085b0542e8e6a08c7eb176d/packages/@ember/-internals/glimmer/tests/integration/application/rendering-test.js#L976-L1025

Is failing during cleanup of test harness. It presents in the same way as https://github.com/emberjs/ember-test-helpers/issues/768.